cmd/kaspawallet/keys/create.go

```diff
@@ -54,23 +57,23 @@ func createKeyPairsFromFunction(numKeys uint32, keyPairFunction func(keyIndex ui
 		return nil, nil, errors.New("Passwords are not identical")
 	}
 
-	encryptedPrivateKeys = make([]*EncryptedPrivateKey, 0, numKeys)
-	for i := uint32(0); i < numKeys; i++ {
-		privateKey, publicKey, err := keyPairFunction(i)
+	encryptedPrivateKeys = make([]*EncryptedMnemonic, 0, len(mnemonics))
+	for _, mnemonic := range mnemonics {
+		extendedPublicKey, err := libkaspawallet.MasterPublicKeyFromMnemonic(params, mnemonic, isMultisig)
```

Comment:
So now we save the mnemonics in the DB?
Is there any reason why it is necessary?
why not internally only use extended keys and only in backup/recover use mnemonics?
This will drastically decrease the attack vector that bip39 imposes

Reply:
Good point